application/helpers/SurveyRuntimeHelper.php

```diff
@@ -422,7 +422,7 @@ public function run($surveyid, $args)
         $this->aSurveyInfo['jPopup'] = json_encode($aPopup);
         $this->aSurveyInfo['mandSoft'] = isset($this->aMoveResult['mandSoft']) ? $this->aMoveResult['mandSoft'] : false;
         $this->aSurveyInfo['mandNonSoft'] = isset($this->aMoveResult['mandNonSoft']) ? $this->aMoveResult['mandNonSoft'] : false;
-        $this->aSurveyInfo['mandViolation'] = $this->aStepInfo['mandViolation'] && $this->okToShowErrors;
+        $this->aSurveyInfo['mandViolation'] = ($this->aStepInfo['mandViolation'] ?? false) && $this->okToShowErrors;
```

Comment:
Not related to PR description. Issue to be reported ?

Reply:
During the tests there were some surveys whose mandViolation was not set. We fixed it as part of the epic that targets the Cloud environment, but we separated from that everything that was possible to separate to avoid having issues with merge conflicts in the future as far as possible. This is why we created this PR and if the mandViolation fix is to be separated from this, then we can remove it from here.